user_docs/en/userGuide.t2t

```diff
@@ -519,6 +518,26 @@ When the menu comes up, You can use the arrow keys to navigate the menu, and the
 | Report clipboard text | NVDA+c | Reports the Text on the clipboard if there is any. |
 %kc:endInclude
 
+++ Speech modes ++[SpeechModes]
+
+There are four speech modes available.

```

Comment:
Since this is the first time speech modes have been discussed at all, jumping right into it without some introduction feels a bit bumpy. Perhaps something like this?

```suggestion
The speech mode, governs how screen content, notifications, responses to commands, and other output, is spoken during operation of NVDA.
The default mode is "talk", and provides for the kind of speech output that is usually expected when using a screen reader.
However, under certain circumstances, or when running particular programs, you may find one of the other speech modes valuable.

The four available speech modes are:
```

Reply:
Very nice and relevant suggestion indeed @XLTechie. Many thanks to you for all your review work and suggestions on this PR.